noty-api/application/src/dev/shreyaspatil/noty/api/controller/NotesController.kt

```diff
@@ -97,6 +98,21 @@ class NotesController @Inject constructor(private val noteDao: NoteDao) {
         }
     }
 
+    fun pinNote(user: User, noteId: String, pinRequest: PinRequest): NoteResponse {
```

Comment:
@mrfamouskk7 Can we rename it to `updateNotePin()`? Because `pinNote` sounds like we are only pinning it and not un-pinning.